### DISCUSSION TITLE
Snapenhance

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [LSPosed/LSPatch#263] [General]
Is there a way to make this work with Snapenhance

**Comment by JasonKhew96** [comment]
https://github.com/rhunk/SnapEnhance/blob/dev/README.md
> Although using this in an unrooted enviroment using something like LSPatch should be working fine, it is not recommended to do so, use at your own risk!

**Reply by pizzav-xyz** [reply]
How do I make sure Snapchat is in scope

**Comment by pizzav-xyz** [comment]
> https://github.com/rhunk/SnapEnhance/blob/dev/README.md
> > Although using this in an unrooted enviroment using something like LSPatch should be working fine, it is not recommended to do so, use at your own risk!

I did portable and embedded snapenhance but I don't see the setting button

**Comment by pizzav-xyz** [comment]
> > https://github.com/rhunk/SnapEnhance/blob/dev/README.md
> > > Although using this in an unrooted enviroment using something like LSPatch should be working fine, it is not recommended to do so, use at your own risk!
> 
> I did portable and embedded snapenhance but I don't see the setting button

@JasonKhew96
